server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/QualityProfileMapper.java

```diff
@@ -32,12 +32,18 @@
 
   void delete(int id);
 
+  /**
+   * @deprecated provide organization
+   */
+  @Deprecated
   List<QualityProfileDto> selectAll();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)